### PR TITLE
first draft of theme README

### PR DESCRIPTION
**Pull request by gwijthoff**
Elements that will need detail are marked in bold. Feel free to edit as needed, @thatbudakguy @rlskoeser @gissoo.

### CODE REVIEW

File: themes/startwords/README.md [hunk 1]
```diff
@@ -1,31 +1,28 @@
-# Blank
+# <div align="center"><img src="https://startwords.cdh.princeton.edu/logotype.svg" /> Theme</div>
```
Comment (rlskoeser) [inline themes/startwords/README.md]: Image should have an alt text here (or maybe split out the logo and the theme label?).

Maybe use markdown image syntax? `Alt`

File: themes/startwords/README.md [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
 
-Blank — starter [Hugo](https://gohugo.io/) theme for developers. Use it to make your own theme.
+[![License: CC BY-NC-SA 4.0](https://img.shields.io/badge/License-CC%20BY--NC--SA%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc-sa/4.0/)
```
Comment (rlskoeser) [inline themes/startwords/README.md]: The CC license will be for the content, but not the theme/code; I think @thatbudakguy suggested Apache 2 license, which we're using for our other code.

File: themes/startwords/README.md [hunk 3]
```diff
@@ -1,31 +1,28 @@
...
+[![Hugo](https://img.shields.io/badge/hugo-0.67-blue.svg)](https://gohugo.io)
 
-[Live Demo](https://themes.gohugo.io/theme/blank/)
+The Startwords Hugo theme is designed for [a journal of the same name](https://startwords.cdh.princeton.edu/) by the [Center for Digital Humanities](cdh.princeton.edu/) at Princeton University.
 
-![Blank theme screenshot](https://github.com/Vimux/blank/blob/master/images/splash.png)
+## Goals
 
-## Installation
+The main architectural goals of this theme are to:
```
Comment (rlskoeser) [inline themes/startwords/README.md]: I'm not sure all of these goals apply to the theme.. (e.g. the choice of static site generator applies to _Startwords_ more generally but not to the theme)

File: themes/startwords/README.md [hunk 4]
```diff
@@ -1,31 +1,28 @@
...
-For more information read the official [quick start guide](https://gohugo.io/getting-started/quick-start/) of Hugo.
-
-## Contributing
-
-Have you found a bug or got an idea for a new feature? Feel free to use the [issue tracker](https://github.com/Vimux/blank/issues) to let me know. Or make directly a [pull request](https://github.com/Vimux/blank/pulls).
-
-## License
-
-This theme is released under the [MIT license](https://github.com/Vimux/blank/blob/master/LICENSE).
+![](https://startwords.cdh.princeton.edu/Information%20Architecture.svg)
```
Comment (rlskoeser) [inline themes/startwords/README.md]: I was going to ask if the architecture diagram belongs here, but I looked at it again and it does seem like our site architecture is pretty important for understanding the content types and the way we've set them up.

File: themes/startwords/README.md [hunk 5]
```diff
@@ -1,31 +1,28 @@
...
-theme = "blank"
-```
-
-For more information read the official [quick start guide](https://gohugo.io/getting-started/quick-start/) of Hugo.
-
-## Contributing
-
-Have you found a bug or got an idea for a new feature? Feel free to use the [issue tracker](https://github.com/Vimux/blank/issues) to let me know. Or make directly a [pull request](https://github.com/Vimux/blank/pulls).
-
-## License
```
Comment (rlskoeser) [inline themes/startwords/README.md]: Shouldn't we have a license section?

File: themes/startwords/README.md [hunk 6]
```diff
@@ -1,31 +1,28 @@
...
+- Layouts: custom content types include feature articles [available under [layouts/article](https://github.com/Princeton-CDH/startwords/tree/master/themes/startwords/layouts/article)], single issues, and list of issues [both available under [layouts/issue](https://github.com/Princeton-CDH/startwords/tree/master/themes/startwords/layouts/issue)].
+- To retain the simplest directory structure that will give us the URLs we want, articles are placed in an issue number directory and have `type:article` specified in the page metadata.
+- Excerpts of the current issue's opening lines are generated on the homepage using Hugo's `<!--more-->` [content summary divider](https://gohugo.io/content-management/summaries/)
+- Article DOIs (registered with [Zenodo](zenodo.org/)) are specified in each article's YAML header, so that article metadata can be easily harvested by [Zotero](https://www.zotero.org/) citation management software.
+- Article footnotes are generated as **contextual notes** using…
```
Comment (rlskoeser) [inline themes/startwords/README.md]: The current plan is to still use markdown footnotes, but maybe we update this once we have it working.

File: themes/startwords/README.md [hunk 7]
```diff
@@ -1,31 +1,28 @@
...
+- Article illustration capabilities are built in using [IIIF](https://iiif.io/) for images and [Sketchfab](https://sketchfab.com/) for 3D embeddings.
```
Comment (rlskoeser) [inline themes/startwords/README.md]: @gwijthoff what do you think about documenting our custom shortcodes with the theme? Any opinion on whether it should it be here or in a readme in the shortcode folder?

**Comment by rlskoeser** [comment]
One more comment I forgot to include in my first notes. @gwijthoff I wondered about including a directory tree style listing of either the theme contents or the site contents that use the theme? Seems like it could be a useful way to document the structure in the readme.

File: themes/startwords/README.md (hunk 3 above)
Comment (thatbudakguy) [reply themes/startwords/README.md]: agree that this one specifically is a goal for startwords but redundant here

File: themes/startwords/README.md (hunk 5 above)
Comment (thatbudakguy) [reply themes/startwords/README.md]: I think we can just include the actual LICENSE text file.

File: themes/startwords/README.md (hunk 7 above)
Comment (thatbudakguy) [reply themes/startwords/README.md]: I like documenting them here!

File: themes/startwords/README.md (hunk 1 above)
Comment (gwijthoff) [reply themes/startwords/README.md]: @rlskoeser I was thinking of simply editing the readme file in the `theme-readme-GW1` branch and then clicking "resolve conversation" for the relevant parts of the conversation in this PR. At least for the elements that I can resolve right now. Does that work?

Comment (rlskoeser) [reply themes/startwords/README.md]: That sounds like a good workflow to me.

**Comment by rlskoeser** [comment]
Oh, also — I think we should be able to use relative links for several of the things that are currently absolute links. 
See https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files

**Comment by gwijthoff** [comment]
@rlskoeser I just incorporated all of your suggestions that I was able to in [this commit](https://github.com/Princeton-CDH/startwords/pull/54/commits/7142cbf62f95d7b7f76fa1afa39e2348a7e7ae82).

@thatbudakguy @gissoo not sure that I have enough context or expertise anymore to document the site's architecture and design, given that so much has happened since I last touched the site's code. Willing to help however I can, though.

There are a couple ideas that @rlskoeser flags in comments above that I wasn't able to address.

- [document custom shortcodes](https://github.com/Princeton-CDH/startwords/pull/54/files/721c3edbf459e8f5868e90bdaf9ee25bb2e8ca01#r397403425)
- [include a LICENSE text file](https://github.com/Princeton-CDH/startwords/pull/54/files/721c3edbf459e8f5868e90bdaf9ee25bb2e8ca01#r397398974)

**Comment by gwijthoff** [comment]
Slightly confused about the relationship between this issue #54 and #48... Think I may have zigged where I should have zagged. 🤷‍♂️

**Comment by rlskoeser** [comment]
@gwijthoff @thatbudakguy have added documentation for our custom shortcodes and linked the apache 2 license. Please review! (@gwijthoff it won't let me add you as a reviewer, I guess because you own the PR.)

**Comment by gwijthoff** [comment]
Excellent, looks great to me @rlskoeser !

**Pull request merged**